db/sql/10_msar_joinable_tables.sql

```diff
@@ -131,8 +131,37 @@ SELECT * FROM output_cte;
 $$ LANGUAGE sql;
 
 
+DROP FUNCTION IF EXISTS msar.get_joinable_tables(integer, oid);
 CREATE OR REPLACE FUNCTION
 msar.get_joinable_tables(max_depth integer, table_id oid) RETURNS
-  SETOF msar.joinable_tables AS $$
-  SELECT * FROM msar.get_joinable_tables(max_depth) WHERE base=table_id
-$$ LANGUAGE sql;
+jsonb AS $$
+DECLARE
+  joinable_tables jsonb;
+  target_table_info jsonb;
+BEGIN
+  CREATE TEMP TABLE jt_cte AS 
```

Comment:
Why make this a temp table rather than a proper CTE here? It's
- slightly less performant under some conditions (i.e., when it's small, and
- affects the volatility setting, meaning it's slower for callers